test/lib/browser/commands/assert-view/index.js

```diff
@@ -241,6 +241,21 @@ describe('assertView command', () => {
             );
         });
 
+        it('should compare images with given tolerance', async () => {
+            const config = mkConfig_({tolerance: 100, antialiasingTolerance: 200, compareOpts: {stopOnFirstFail: true}});
+            const browser = stubBrowser_(config);
+
+            browser.prepareScreenshot.resolves({canHaveCaret: 'foo bar', pixelRatio: 300});
+
+            await browser.publicAPI.assertView(null, null, {tolerance: 500});
+
+            assert.calledOnceWith(
+                Image.compare,
+                sinon.match.any, sinon.match.any,
+                {canHaveCaret: 'foo bar', tolerance: 500, antialiasingTolerance: 200, pixelRatio: 300, compareOpts: {stopOnFirstFail: true}}
```

Comment:
you do not need to copy-paste whole previous test. You should test only tolerance overwriting here:
```js
const config = mkConfig_({tolerance: 100});
...
assert.calledOnceWith(
    Image.compare,
    sinon.match.any,
    sinon.match.any,
    sinon.match({tolerance: 500})
);
```